src/e2e/pages/landingPage.js

```diff
@@ -70,9 +86,21 @@ export class LandingPage {
     await this.firefoxLogo.click()
   }
 
+  async heroSectionTexts(){
+    return {
+      heroHeaderText: "Find out if your personal information has been compromised",
+      heroDescriptionText: "Stay safe with privacy tools from the makers of ⁨Firefox⁩ that protect you from hackers and companies that publish and sell your personal information. We’ll alert you of any known data breaches, find and remove your exposed info and continually watch for new exposures.",
+      heroEmailInputPlaceholder: "Enter email address",
+    }
+  }
+
   async checkBanner(){
-    if(await this.falseDoorBanner.isVisible()){
-      await this.falseBannerCloseButton.click()
+    await this.page.waitForSelector('//div[starts-with(@class, "FalseDoorBanner_falseDoorTest")]',{
```

Comment:
Q: Is there a way to do this without xpath (since I dont think we use them elsewhere in selectors/locators)?

Reply:
this is a temp banner to my knowledge, this will be removed as soon as that banner is removed.